db_test.go

```diff
@@ -743,28 +743,44 @@ func TestWALFlushedOnDBClose(t *testing.T) {
 }
 
 func TestWALSegmentSizeOption(t *testing.T) {
-	options := *DefaultOptions
-	options.WALSegmentSize = 2 * 32 * 1024
-	db, delete := openTestDB(t, &options)
-	defer delete()
-	app := db.Appender()
-	for i := int64(0); i < 155; i++ {
-		_, err := app.Add(labels.Labels{labels.Label{Name: "wal", Value: "size"}}, i, rand.Float64())
-		testutil.Ok(t, err)
-		testutil.Ok(t, app.Commit())
+	tests := map[int]func(dbdir string, options Options){
+		2 * 32 * 1024: func(dbDir string, options Options) {
```

Comment:
some comments for each test case would be a nice reminder.

// Custom wal size.

// Default wal size.

// WAL is disabled.